deno_dist/utils/cloudflare.ts

```diff
@@ -1,3 +1,5 @@
+/// <reference types="@cloudflare/workers-types" />
```

Comment:
Wouldn't type import work as well:

```typescript
import type { KVNamespace } from '@cloudflare/workers-types'
```

It would prevent Cloudflare types from being added to the TS compilation context.

Reply:
I see. But this file in `deno_dist` is used for Deno. So, it is not needed. I've removed it.